docs/modules/deployment/pages/time-series-storage/timeseries/cortex.adoc

```diff
@@ -28,19 +28,23 @@ If your aim is to get up to speed quickly in a test environment, you can use Doc
 
 . Download the plugin's `.kar` file from GitHub (https://github.com/OpenNMS/opennms-cortex-tss-plugin/releases).
 . Copy the file into `$\{OPENNMS_HOME}/deploy`.
-. Enable and configure time series database plugins in {page-component-title}:
+. Enable and configure time series database plugins in `$\{OPENNMS_HOME}/etc/opennms.properties.d/timeseries.properties`:
 +
 [source, properties]
 ----
-org.opennms.timeseries.strategy=integration
-org.opennms.timeseries.tin.metatags.tag.node=${node:label}
+org.opennms.timeseries.strategy=integration <1> <2>
+org.opennms.timeseries.tin.metatags.tag.node=${node:label} <3>
 org.opennms.timeseries.tin.metatags.tag.location=${node:location}
 org.opennms.timeseries.tin.metatags.tag.geohash=${node:geohash}
 org.opennms.timeseries.tin.metatags.tag.ifDescr=${interface:if-description}
 org.opennms.timeseries.tin.metatags.tag.label=${resource:label}
 ----
+<1> Make sure that the `org.opennms.timeseries.strategy` property is not set in `$\{OPENNMS_HOME}/etc/opennms.properties` or any other file in `$\{OPENNMS_HOME}/etc/opennms.properties.d/`.
```

Comment:
This only applies for dual-writes. When *not* using dual-writes, this *must* be set.

Reply:
Intention here is to make sure the setting is only defined in one place. Need to clarify that.

Reply:
```suggestion
<1> Make sure that the `org.opennms.timeseries.strategy` property is only set once so you don't have conflicting values in multiple `properties` files.
```